cmake/NanoBindMinimal.cmake

```diff
@@ -0,0 +1,16 @@
+# =============================================================================
+# minimal nanobind interface
+# =============================================================================
+
+set(NB_DIR ${PROJECT_SOURCE_DIR}/external/nanobind)
+
+function(make_nanobind_target TARGET_NAME PYINC)
+  add_library(${TARGET_NAME} STATIC ${NB_DIR}/src/nb_combined.cpp)
+  target_include_directories(${TARGET_NAME} SYSTEM PUBLIC ${NB_DIR}/include)
+  target_include_directories(${TARGET_NAME} SYSTEM PRIVATE ${NB_DIR}/ext/robin_map/include ${PYINC})
+  target_compile_options(${TARGET_NAME} PRIVATE -fno-strict-aliasing)
+  target_compile_definitions(${TARGET_NAME} PRIVATE -D_CRT_SECURE_NO_WARNINGS)
```

Comment:
This seems to be a Microsoft thing. Do we need it on Linux?

Reply:
@ferdonline same here?

Reply:
See comment above